projects/medical/2d_image/fundus_photography/orvs/tools/prepare_dataset.py

```diff
@@ -0,0 +1,55 @@
+import glob
+import os
+
+import numpy as np
+from PIL import Image
+
+root_path = 'data/'
+img_suffix = '.jpg'
+seg_map_suffix_list = ['.jpg', '.png', '.tif']
+save_img_suffix = '.png'
+save_seg_map_suffix = '.png'
+
+x_train = glob.glob(
+    os.path.join('data/Vessels-Datasets/*/Train/Original/Images/*' +
+                 img_suffix))
+x_test = glob.glob(
+    os.path.join('data/Vessels-Datasets/*/Test/Original/Images/*' +
+                 img_suffix))
+
```

Comment:
I found it contains other datasets
https://github.com/AbdullahSarhan/ICPRVessels